launcher.js

```diff
@@ -0,0 +1,45 @@
+// A "launcher" to start the (Node version of the) router from the command line.
```

Comment:
OK, in reviewing with Antranig, it looks like fluid.require can't be used from index.js in this case.  require definitely works, we just need to confirm that there are no side effects.

I will manually test both require("flocking-midi-router") and fluid.require("%flocking-midi-router") in another package.